src/Symfony/Component/Serializer/Normalizer/GetSetMethodNormalizer.php

```diff
@@ -47,7 +47,7 @@ public function denormalize($data, $class, $format = null, array $context = arra
         $normalizedData = $this->prepareForDenormalization($data);
 
         $reflectionClass = new \ReflectionClass($class);
-        $object = $this->instantiateObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes);
+        $object = $this->instantiateComplexObject($normalizedData, $class, $context, $reflectionClass, $allowedAttributes, $format);
```

Comment:
>Take a look at my example: the new attribute is commented and fetched thanks to php functions so there is no bc problems.

@Ener-Getick the problem I was showing was not a BC break introduced by the function signature, but by the fact that in this example `$myParam` cannot be taken as `$format`.